### PR TITLE
add Domain.get_default_timezone() for an actual pytz object

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by dannyroberts** [comment]
@nickpell

File: corehq/apps/domain/models.py [hunk 1]
```diff
@@ -363,6 +363,11 @@ def wrap(cls, data):
             self.save()
         return self
 
+    def get_default_timezone(self):
+        """return a timezone object from self.default_timezone"""
+        import pytz
```
Comment (nickpell) [inline corehq/apps/domain/models.py]: any particular reason for the inline import?

Comment (dannyroberts) [inline corehq/apps/domain/models.py]: not really than that I expect this to be somewhat short-term and it's only called in one place. I can move it to the top. One sec

Comment (nickpell) [inline corehq/apps/domain/models.py]: if short term than no big deal

**Comment by dannyroberts** [comment]
haha ok never mind :)

**Pull request merged**